app/_src/mesh/features/openshift-quickstart.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Get started with Red Hat OpenShift and Kong Mesh
+content_type: tutorial
+description: This guide explains how to get started on Kong Mesh with Red Hat OpenShift.
+---
+
+This is a quickstart to get you up and running with Kong Mesh in standalone mode on OpenShift.
+
+This tutorial does not require a license, Kong Mesh can start in evaluation mode, and limits you 5 dataplanes/sidecars to use. Just enough dataplanes to get comfortable with the product.
+
+For OpenShfit, we'll be spinning up a ROSA 4.12 cluster. This tutorial does also assume some base level knowledge of OpenShift.
+
+This tutorial will cover:
+
+* How to use the Red Hat Certified Kong Mesh Images
+* How to implement the required OpenShift SecurityContextConstraints (SCC) for the kong-mesh sidecar
+* Deploy and join the Kong for Kubernetes Ingress Controller (KIC) to the mesh
```

Comment:
Why do we need to deploy KIC in this guide?

Reply:
We deploy KIC so that the user can reach those microservice from curl or the browser and validate visually everything is in working order.